.github/workflows/tests.yml

```diff
@@ -64,5 +65,11 @@ jobs:
             ~\AppData\Roaming\LJ4W
             ~\AppData\Roaming\luarocks
           key: ${{ runner.os }}-tests
+
+      - name: Install NPM dependencies
+        uses: chill-viking/npm-ci@latest
```

Comment:
Can we change that to the "official" node action, here: https://github.com/actions/setup-node

Reply:
Sure.  Didn't see there was an official one.

Reply:
I keep on getting an error with this action:
![image](https://github.com/user-attachments/assets/2c64e9d6-3d92-408a-8bb3-2901037d3e9c)

Not too sure what this is about, but feels like npm does not get installed or isn't found.

Reply:
I will revert back to the original action.
Btw, do you know why win CI is so damn slow?

Reply:
@gorillamoe  I have reverted the changes to action and will make a separate PR